apps/docs/docs/learn/04-styling-ui/02-using-styles.mdx

```diff
@@ -69,7 +69,7 @@ The actual implementation is optimized for performance.
 
 Styles can be applied conditionally at runtime by using common JavaScript
 patterns such as ternary expressions and the `&&` operator. `stylex.props`
-ignores falsy values such as `null`, `undefined` or `false`.
+ignores falsy values such as `null`, `undefined` or `false` and empty objects.
```

Comment:
Please revert this change. We don't want people passing in empty objects into `stylex.props`. There is no use-case and it'll just confuse people.